.github/workflows/winget-submission.yml

```diff
@@ -1,9 +1,14 @@
 name: Submit Nushell package to Windows Package Manager Community Repository
 
 on:
-  workflow_dispatch:
   release:
-    types: [published]
+    types: [released]
```

Comment:
How will this interact with draft releases?

Reply:
I think the difference between the `published` and `released` trigger is that `released` does not apply to pre-releases and drafts, so I'd thought that would make more sense here.

Reply:
Ok sounds good! Yeah we don't want to ship if it is still in the draft stage, in case we notice a breakage.
Ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
> Yeah we don't want to ship if it is still in the draft stage,

I'm not even sure how is that supposed to work. Draft releases are not public.